x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/constants.ts

```diff
@@ -293,52 +314,122 @@ export const SAVE_CUSTOM_VISUAL_WALKTHROUGH_TITLE = i18n.translate(
   }
 );
 
+export const SAVE_CUSTOM_VISUAL_WALKTHROUGH_LINK = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSource.saveCustom.visualWalkthrough.link',
+  {
+    defaultMessage: 'Check out the documentation',
+  }
+);
+
 export const SAVE_CUSTOM_STYLING_RESULTS_TITLE = i18n.translate(
   'xpack.enterpriseSearch.workplaceSearch.contentSource.saveCustom.stylingResults.title',
   {
     defaultMessage: 'Styling Results',
   }
 );
 
+export const SAVE_CUSTOM_STYLING_RESULTS_LINK = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSource.saveCustom.stylingResults.link',
+  {
+    defaultMessage: 'Display Settings',
```

Comment:
Should this be consistent with other casing?
```suggestion
    defaultMessage: 'Display settings',
```

Reply:
I'd prefer not to change this in this PR for 2 reasons:
1) This is in line with the casing used in other parts of the app:
![image](https://user-images.githubusercontent.com/11838280/133434036-49477d0b-c18a-41c3-968e-330293833fd5.png)
2) If sentence case is preferred everywhere, I think it should be part of a bigger Copy review initiative.